apm-sniffer/apm-sdk-plugin/pulsar-plugin/src/main/java/org/apache/skywalking/apm/plugin/pulsar/PulsarConsumerInterceptor.java

```diff
@@ -54,12 +54,11 @@
     @Override
     public void beforeMethod(EnhancedInstance objInst, Method method, Object[] allArguments, Class<?>[] argumentsTypes,
                              MethodInterceptResult result) throws Throwable {
-        if (allArguments.length > 0 && allArguments[0] != null) {
+        if (allArguments[0] != null) {
             ConsumerEnhanceRequiredInfo requiredInfo = (ConsumerEnhanceRequiredInfo) objInst.getSkyWalkingDynamicField();
-            requiredInfo.setStartTime(System.currentTimeMillis());
             AbstractSpan activeSpan = ContextManager.createEntrySpan(OPERATE_NAME_PREFIX +
                     requiredInfo.getTopic() + CONSUMER_OPERATE_NAME + requiredInfo.getSubscriptionName(), null)
-                    .start(requiredInfo.getStartTime());
+                    .start(System.currentTimeMillis());
```

Comment:
`#createEntrySpan` has set start time automatically, you could find it in source does.

Reply:
Yes, AbstractTracingSpan.start() is already set the startTime